client/my-sites/plugins/not-available-error.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import EmptyContent from 'components/empty-content';
+import DocumentHead from 'components/data/document-head';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+
+function NotAvailableError( { translate } ) {
+	return (
+			<Main>
+				<DocumentHead title={ translate( 'Plugins' ) } />
+				<SidebarNavigation />
+				<EmptyContent
+					title={ translate( 'Not Available' ) }
+					line= { translate( 'The page you requested could not be found' ) }
```

Comment:
Hi @michelleweber, @rickybanister,
I checked the conditions in the code and I'm able to confirm that this error message shows when the user has not the right role/permissions as @michelleweber referred at the start. To get the right permissions the user needs to get in touch with the site administrator.

The message may not be specific to some site. If only one site is selected the message appears if the user has not the right permission/role on that site so its specific to that site. But If no site is selected the message appears if the user has not the right role/permissions for any of the sites e.g. it needs to have right permissions for at least one site in order for this error message not appear. So the message is not specific to a site in this case.  Off course if needed this could be divided and show 2 different error messages.